tools/rpi-otp-private-key

```diff
@@ -134,7 +155,8 @@ if [ $(((0x$BOARD_INFO >> 12) & 15)) = 3 ]; then
 elif [ $(((0x$BOARD_INFO >> 12) & 15)) = 4 ]; then
     MAX_ROW_COUNT=16
 else
-    die "Chip not supported"
+   echo "WARNING: Secure-boot is only supported on Pi4 and newer models"
```

Comment:
Indent is a bit off here? Is there a reason why we allow this to work for unsupported models now?

Reply:
done